library/smarty/plugins/modifier.xla.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
neat stuff!
smarty! smarty! smarty!

![2DV](https://user-images.githubusercontent.com/278968/126725796-4beac087-cd9c-490f-97fc-049e0e4207a9.gif)

Reply:
I debated on switching it all out for twig during the refactor, since supposedly Smarty is supposed to be deprecated, but it would have been a much bigger project due to the growth charts and all of the javascript.  It should be easier now that a lot of the redundant code is removed... but still a project for another day.

Reply:
I should clarify, that I saw in the OpenEMR wiki that Smarty is supposed to be deprecated.

Reply:
We are stuck with Smarty on the php-gacl code, but definitely need to get to a "modern" smarty version for that:
https://github.com/openemr/openemr/pull/3049

And then either do same or get out of Smarty for the forms, core (practice settings and some things like rx), and the calendar.

Reply:
Interesting... I looked at that php-gacl code and everything looks pretty compatible w/ twig.  Looking at that issue, is the only thing outstanding the assign_by_ref functionality?

Reply:
Going to twig would be very cool! But likely lots of tedious work in php-gacl to do that. i think that PR only addressed php-gacl to smarty3; still need to do core, forms, and calendar.